2_git/week1/my_favorite_food.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hi, this file is not necessary! Please, remove it and push again :)

Reply:
Hi Paula, I have pushed it again and the terminal shows that its being pushed but I dont see it on git. Can you?
<img width="979" alt="Screenshot 2023-08-25 at 13 02 48" src="https://github.com/HackYourFuture-CPH/class26-homework/assets/115549210/eea25028-e297-4a38-8dd1-a0260fd30e7c">

Reply:
Please check again